api/main.go

```diff
@@ -157,6 +158,20 @@ func PricePlanFile(ctx context.Context, jsonBlob string) (float64, []string, []s
 					Count:     change.Change.After.(map[string]interface{})["sku"].([]interface{})[0].(map[string]interface{})["capacity"].(float64),
 					Size:      change.Change.After.(map[string]interface{})["sku"].([]interface{})[0].(map[string]interface{})["name"].(string),
 					Location:  change.Change.After.(map[string]interface{})["location"].(string),
+					//TODO make helpers to grab a key of desired type from chage struct
+				})
+				break
+			case "azurerm_managed_disk":
+				size, err := strconv.ParseInt(change.Change.After.(map[string]interface{})["disk_size_gb"].(string), 10, 64)
```

Comment:
If you want you can convert the field to a `float64`. `int` was only cause that's how I was thinking